### ISSUE TITLE
Document that mavenCentral() must be used to resolve Boot's Gradle plugin

### ISSUE THREAD

**Issue by wilkinsona** [spring-projects/spring-boot#26138]
We used to publish our Gradle plugin to Gradle's Plugin Portal via Bintray. This allowed us to stage a release to https://repo.spring.io, smoke test it, and then promote it to Bintray and the Portal if those tests passed. Unfortunately, following Bintray being sunset, the Gradle team are not (yet?) going to provide an equivalent publishing method. The only option currently available is to publish directly from a Gradle build. This is a non-starter for us as it would prevent us from smoke testing a release before its artifacts become public.

In short, new releases will not be available from Gradle's Plugin Portal. We need to update our documentation to describe how to use `mavenCentral()` for plugin resolution.

**Comment by philwebb** [comment]
We're going to manually upload it for this release.
